meta/labs/README.md

```diff
@@ -3,6 +3,24 @@
 This document will contain a brief overview of our labs program and what it'll
 entail going into the 2015 fall semester.
 
+## Format
+
+The Labs program will follow this format:
+
+1. Workshops
+   - These are based on the workshops from Hack Camp
```

Comment:
I know that we didn't discuss the above ^, but club leaders are asking me if they can create their own workshops, someone proposed an Agar.io one and I thought that sounded awesome!